src/TranslationValidator.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace CrazyFactory\Translations;
+
+
+class TranslationValidator
+{
+    public static function isValidKey(string $key): bool
+    {
+        $re = '/^[a-z0-9]*([._][a-z0-9]*)*$/m';
+
+        return preg_match($re, $key);
+    }
+
+    public static function isValidValue(string $value): bool
+    {
+        $re = '/^[{<].*[>}]/m';
+
+        return !preg_match($re, $value);
+    }
+
+    public static function isValidLocale(string $locale): bool
+    {
+        $languages = ['cs', 'de', 'en-GB', 'en-US', 'es', 'fr', 'hr', 'it', 'nb', 'nl', 'pl', 'pt', 'fi', 'sv'];
```

Comment:
The valid languages should come from a configuration instead being hard-coded.
You have at least 2 options:
- `config.local.php`
- database

I think the smartest way is to provide a configuration array for the `TranslationValidator`, which has the values for `ValidLocale`, `ValidTranslationsState`, `ValidTranslationRevisionsState`.
This way we keep it testable and can easily add/remove languages/states without the need to change the code or make a new release.